automatic/mumble/update.ps1

```diff
@@ -19,27 +19,21 @@ function global:au_SearchReplace {
 }
 
 function global:au_AfterUpdate {
-  Update-metadata -key "releaseNotes" -value $Latest.ReleaseNotes
+  Update-Metadata -key "releaseNotes" -value $Latest.ReleaseNotes
 }
 
 function global:au_GetLatest {
+  $LatestRelease = Get-GitHubRelease mumble-voip mumble
   $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-
-  $re = '\.msi$'
-  $url32 = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
-
-  $verRe = '\/'
-  $version32 = $url32 -split "$verRe" | select -last 1 -skip 1
-
-  $releaseNotes = $download_page.Links | ? href -match "mumble.info\/blog" | select -first 1 -expand href
-
+  $url32 = $LatestRelease.assets | Where-Object {$_.name.StartsWith("mumble_client")} | Where-Object {$_.name.EndsWith(".msi")} | Select-Object -ExpandProperty browser_download_url
+  $releaseNotes = $download_page.Links | ? href -match "www.mumble.info\/blog" | Select-Object -first 1 -expand href
   if (!$releaseNotes) {
     $releaseNotes = "$releases/tag/$version32"
```

Comment:
It looks like the `$version32` variable definition was removed, but it is still used here.

Reply:
alright, i'll remove it

Reply:
In the case that the release notes is null/empty, it looks like there is not handling for it anymore. Instead of removing that if statement entirely, the code should be updated to still link to that page; so the `$version32` would still be needed unless you used a different way to get the fallback release notes url.

Reply:
Ok, I've used `$LatestRelease.body` as a fallback option.